<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Devices settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>kitty.png</normaloff>kitty.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilo general para la aplicación */
QWidget {
    background-color: #1e1e1e; /* Fondo oscuro */
    color: #d4d4d4; /* Texto claro */
    font-family: 'Segoe UI', sans-serif; /* Fuente moderna de Windows */
    font-size: 12px;
}

/* Botones (QPushButton) */
QPushButton {
    background-color: #3a3d41; /* Fondo oscuro para botones */
    color: #ffffff; /* Texto blanco */
    border: none;
    border-radius: 6px; /* Esquinas redondeadas */
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #505357; /* Fondo más claro al pasar el ratón */
}
QPushButton:pressed {
    background-color: #292b2f; /* Fondo más oscuro al presionar */
}

/* Campos de texto (QLineEdit, QPlainTextEdit, QTextEdit) */
QLineEdit, QPlainTextEdit, QTextEdit {
    background-color: #252526; /* Fondo oscuro */
    color: #d4d4d4; /* Texto claro */
    border: 1px solid #3c3c3c; /* Borde gris */
    border-radius: 6px;
    padding: 4px;
}
QLineEdit:focus, QPlainTextEdit:focus, QTextEdit:focus {
    border: 1px solid #0078d4; /* Borde azul al enfocar */
}

/* Tablas (QTableWidget, QTreeWidget) */
QTableWidget, QTreeWidget {
    background-color: #252526; /* Fondo oscuro */
    border: 1px solid #3c3c3c; /* Borde gris */
    border-radius: 6px;
    gridline-color: #404040; /* Líneas de tabla más claras */
    color: #d4d4d4; /* Texto claro */
}
QTableWidget::item, QTreeWidget::item {
    padding: 8px;
}
QHeaderView::section {
    background-color: #333333; /* Fondo ligeramente más claro */
    color: #d4d4d4; /* Texto claro */
    border: none;
    padding: 8px;
    font-weight: bold;
    text-align: left;
}

/* Scroll en tablas */
QTableWidget QScrollBar:vertical, QTreeWidget QScrollBar:vertical {
    background-color: #2a2a2a; /* Fondo oscuro para barra */
    border: none;
    width: 10px;
}
QTableWidget QScrollBar::handle:vertical, QTreeWidget QScrollBar::handle:vertical {
    background-color: #3a3d41; /* Manija gris */
    border-radius: 5px;
}
QTableWidget QScrollBar::handle:vertical:hover, QTreeWidget QScrollBar::handle:vertical:hover {
    background-color: #505357; /* Manija más clara al pasar el ratón */
}

/* Comboboxes (QComboBox) */
QComboBox {
    background-color: #252526; /* Fondo oscuro */
    color: #d4d4d4; /* Texto claro */
    border: 1px solid #3c3c3c; /* Borde gris */
    border-radius: 6px;
    padding: 4px;
}
QComboBox:hover {
    border: 1px solid #0078d4; /* Borde azul al pasar el ratón */
}
QComboBox::drop-down {
    border: none;
    background-color: #333333; /* Fondo ligeramente más claro */
}

/* Barras de progreso (QProgressBar) */
QProgressBar {
    background-color: #3c3c3c; /* Fondo gris oscuro */
    color: #ffffff; /* Texto blanco */
    border: 1px solid #3c3c3c; /* Borde gris */
    border-radius: 6px;
    text-align: center;
}
QProgressBar::chunk {
    background-color: #0078d4; /* Azul progreso */
    border-radius: 6px;
}

/* Sliders (QSlider) */
QSlider::groove:horizontal, QSlider::groove:vertical {
    background-color: #3c3c3c; /* Canal gris oscuro */
    border: 1px solid #3c3c3c; /* Borde gris */
    height: 8px; /* Ancho del canal */
    border-radius: 4px;
}
QSlider::handle:horizontal, QSlider::handle:vertical {
    background-color: #0078d4; /* Control azul */
    border: 1px solid #005a9e; /* Borde azul oscuro */
    border-radius: 6px;
    width: 14px; /* Tamaño del controlador */
    height: 14px;
}
QSlider::handle:hover {
    background-color: #005a9e;
}

/* Casillas de verificación (QCheckBox) */
QCheckBox {
    spacing: 5px;
}
QCheckBox::indicator {
    width: 16px;
    height: 16px;
    background-color: #252526; /* Fondo oscuro */
    border: 1px solid #3c3c3c; /* Borde gris */
    border-radius: 4px;
}
QCheckBox::indicator:checked {
    background-color: #0078d4; /* Azul al seleccionar */
    border: 1px solid #005a9e; /* Borde azul */
}

/* Pestañas (QTabWidget) */
QTabWidget::pane {
    border: 1px solid #3c3c3c;
    border-radius: 6px;
}
QTabBar::tab {
    background-color: #252526; /* Fondo oscuro */
    color: #d4d4d4; /* Texto claro */
    border: 1px solid #3c3c3c;
    border-radius: 6px;
    padding: 8px;
}
QTabBar::tab:selected {
    background-color: #333333; /* Fondo más claro */
    border: 1px solid #0078d4;
    color: #ffffff; /* Texto blanco */
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #3a3d41;border-radius:5px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-family: 'Segoe UI', sans-serif;
font-size: 16px;</string>
           </property>
           <property name="text">
            <string>Devices settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;border: 1px solid white</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:none; border:none</string>
              </property>
              <item>
               <property name="text">
                <string>Select country</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>US</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2" rowspan="3">
             <widget class="QPushButton" name="pushButton_2">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>70</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;background-color:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>play_9959717.png</normaloff>play_9959717.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">border:none;font-family: 'Segoe UI', sans-serif;
font-size: 14px;</string>
              </property>
              <property name="text">
               <string>Set proxy</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="3">
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>wifi_12180764.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;border: 1px solid white</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="1">
             <widget class="QLabel" name="label_8">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;font-family: 'Segoe UI', sans-serif;
font-size: 14px;</string>
              </property>
              <property name="text">
               <string>Required applications</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="3">
             <widget class="QPushButton" name="pushButton_4">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>70</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;background-color:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>play_9959717.png</normaloff>play_9959717.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="3">
             <widget class="QLabel" name="label_7">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>apps_12180708.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: 'Segoe UI', sans-serif;
font-size: 18px;</string>
        </property>
        <property name="text">
         <string>Stop all process</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
